### ISSUE TITLE
The narrator announces twice with a message that the TreeNode is collapsed/expanded

### ISSUE THREAD

**Issue by SergeySmirnov-Akvelon** [dotnet/winforms#6936]
### .NET version

7.0.0-preview.4.22178.1

### Did it work in .NET Framework?

No

### Issue description

The narrator announces twice with a message that the `TreeNode` is`Collapsed`/`Expanded`

### Steps to reproduce

1. Create a `Form` with a `TreeView`.
2. Add multiple `TreeNodes` to the `TreeView`.
3. Add multiple `TreeNodes` to the first `TreeNode`.
4. Run the application.
5. Run the `Narrator`.
6. Collapse/Expand first `TreeNode`.
![TreeNodeExpandCollapse](https://user-images.githubusercontent.com/23376742/160808015-9a04ef8f-526f-44a5-9001-1e2b0db1f35b.gif)

**Comment by vladimir-krestov** [comment]
@Olina-Zhang, could you please check this case?
I can't reproduce this in the latest main branch. Please provide more info and repro steps, if you find this bug. If no, please close this Issue.

**Comment by Olina-Zhang** [comment]
@vladimir-krestov Verified this issue in the latest .NEt 7.0 build from main branch, it cannot repro: Narrator just reads collapse/expand state only once, but in NarratorBuddy, it records twice. Can we close this bug?
![image](https://user-images.githubusercontent.com/26474449/170906745-cdb9ebed-e7a2-4417-93c5-177cca479125.png)

**Comment by vladimir-krestov** [comment]
@merriemcgaw, can we close this until getting users requests?
